service/internal/checksum/middleware_setup_context.go

```diff
@@ -47,9 +60,11 @@ type setupOutputContext struct {
 	// GetValidationMode is a function to get the checksum validation
 	// mode of the output payload from the input parameters.
 	//
-	// Given the input parameter value, the function must return the validation
-	// mode and true, or false if no mode is specified.
```

Comment:
Aren't we still returning true or false on the interface? Why remove the comment? If the value is required to prevent changing the interface, we should definitely mention that on the comment